torch/lib/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1555,12 +1555,62 @@ void ProcessGroupNCCL::groupEnd() {
   --ncclActiveGroupCounter_;
 }
 
+#ifdef ENABLE_NCCL_P2P_SUPPORT
```

Comment:
@zasdfgbnm  can you also move the whole code under this #ifdef into the one above w/ alltoall_single, send, recv? 

https://github.com/pytorch/pytorch/blob/1c39d26db63c1c92b90aa269837c749bf94fde57/torch/lib/c10d/ProcessGroupNCCL.cpp#L1478

Reply:
done